.travis.yml

```diff
@@ -1,19 +1,16 @@
 language: python
-sudo: false
 python:
     - 2.7
-    - 3.4
     - 3.5
     - 3.6
+    - 3.7
+    - 3.8
     - pypy
     - pypy3
 matrix:
```

Comment:
The Travis config validator told me yesterday that `matrix` was a deprecated synonym for `jobs`. Having both as keys definitely led to unexpected behaviour (in gevent's complicated config).